docs/assets/operators.json

```diff
@@ -0,0 +1,128 @@
+{
+    "aggregators": {
```

Comment:
Question: Are you using an object where the URLs are the keys to make it extensible in the future? E.g., adding info about who runs it, its location, etc.?